client/header/activity-panel/panels/help.js

```diff
@@ -0,0 +1,290 @@
+		},
+		{
+			title: __( 'Product Shipping Classes', 'woocommerce-admin' ),
+			link: 'https://docs.woocommerce.com/document/product-shipping-classes/?utm_source=help_panel',
+		},
+		showWCS && {
+			title: __( 'WooCommerce Shipping setup and configuration', 'woocommerce-admin' ),
+			link: 'https://docs.woocommerce.com/document/woocommerce-services/#section-3/?utm_source=help_panel',
+		},
+		{
+			title: __( 'Learn more about configuring your shipping settings', 'woocommerce-admin' ),
+			link: 'https://docs.woocommerce.com/documentation/plugins/woocommerce/getting-started/shipping/?utm_source=help_panel',
+		},
+	].filter( Boolean );
+}
+
+function getTaxItems( props ) {
+	const { countryCode } = props;
+	const {
+		automatedTaxSupportedCountries = [],
+		taxJarActivated,
+	} = props.getSetting( 'onboarding', {} );
+
+	const showWCS = (
+		! taxJarActivated && // WCS integration doesn't work with the official TaxJar plugin.
+		automatedTaxSupportedCountries.includes( countryCode )
+	);
+
+	return [
+		{
+			title: __( 'Setting up Taxes in WooCommerce', 'woocommerce-admin' ),
+			link: 'https://docs.woocommerce.com/document/setting-up-taxes-in-woocommerce/?utm_source=help_panel',
+		},
+		showWCS && {
+			title: __( 'Automated Tax calculation using WooCommerce Services', 'woocommerce-admin' ),
+			link: 'https://docs.woocommerce.com/document/woocommerce-services/?utm_source=help_panel#section-10',
+		},
+	].filter( Boolean );
+}
+
+function getItems( props ) {
+	const { taskName } = props;
+
+	switch ( taskName ) {
+		case 'products':
+			return getProductsItems();
+		case 'appearance':
+			return getAppearanceItems();
+		case 'shipping':
+			return getShippingItems( props );
+		case 'tax':
+			return getTaxItems( props );
+		case 'payments':
+			return getPaymentsItems( props );
+		default:
+			return [];
+	}
+}
+
+function handleOnItemClick( props, event ) {
+	const { taskName } = props;
+
+	// event isn't initially set when triggering link with the keyboard.
+	if ( ! event ) {
+		return;
+	}
+
+	props.recordEvent( 'help_panel_click', {
+		task_name: taskName,
+		link: event.currentTarget.href,
+	} );
+}
+
+function getListItems( props ) {
+	const itemsByType = getItems( props );
+	itemsByType.push( {
+		title: __( 'WooCommerce Docs', 'woocommerce-admin' ),
+		link: 'https://docs.woocommerce.com/?utm_source=help_panel',
+	} );
+
+	const onClick = partial( handleOnItemClick, props );
+
+	return itemsByType.map( ( item ) => ( {
+		title: <Text as="div" variant="button">{ item.title }</Text>,
+		before: <Icon icon={ page } />,
+		after: <Icon icon={ chevronRight } />,
+		linkType: 'external',
+		target: '_blank',
+		href: item.link,
+		onClick,
+	} ) );
+}
+
+export const HelpPanel = ( props ) => {
+	const { taskName } = props;
+	useEffect( () => {
+		props.recordEvent( 'help_panel_open', {
+			task_name: taskName,
+		} );
+	}, [ taskName ] );
```

Comment:
Can you use the url to conditionally add `task_name` to the activity panel open event? Otherwise, we're sending two nearly identical events.

https://github.com/woocommerce/woocommerce-admin/blob/41d84f906331bd3c5c1fd1b767a5ee15babd101e/client/header/activity-panel/index.js#L78

Reply:
From the issue: